### ISSUE TITLE
Unable to find Dataset 

### ISSUE THREAD

**Issue by lylia0009** [jbencina/facebook-news#1]
Is the dataset still available? Unable to access it through https://bigquery.cloud.google.com/dataset/jbencina-144002:fb_news

**Comment by jbencina** [comment]
It should be good. I just checked the link and see the data. I also checked
permissions and they are fine. Are you logged into Google while trying to
access? What error are you getting?

**Comment by lylia0009** [comment]
Thanks, let me try again....
